code/app/src/main/java/com/example/bread/fragment/AddMoodEventFragment.java

```diff
@@ -41,6 +52,10 @@ public class AddMoodEventFragment extends Fragment {
     private MoodEventRepository moodEventRepository;
     private ParticipantRepository participantRepository;
     private LocationHandler locationHandler;
+    private ImageButton uploadImage;
+    private ActivityResultLauncher<Intent> resultLauncher;
+    private String imageBase64;
+    private ImageHandler imageHandler;
```

Comment:
Its a static class we don't need a private variable that needs to be initialized.